### PR TITLE
Source Google Adwords: fix CI (#2929)

### LINKED ISSUE #2929: fix Google Adwords CI  & publish new version
## Description
The Google Adwords CI is [failing](https://status-api.airbyte.io/tests/summary/source-google-adwords-singer/). I'm pretty sure nothing is wrong with the connector but it seems like we may have no data in the underlying source. 

Once this has been fixed, we should bump & publish a new version to pick up the changes from https://github.com/airbytehq/airbyte/pull/2918

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1200367912513076/1200368101988801) by [Unito](https://www.unito.io)

### PR DESCRIPTION
**Pull request by Zirochkaa**
Add `tap-adwords==1.13.0` to required packeges in `setup.py` file.

## What
This PR is for #2929 issue.

## Pre-merge Checklist
- [ ] *Run integration tests*
- [ ] *Publish Docker images*

### CODE REVIEW

File: airbyte-integrations/connectors/source-google-adwords-singer/build.gradle [hunk 1]
```diff
@@ -15,6 +15,12 @@ dependencies {
     implementation files(project(':airbyte-integrations:bases:base-singer').airbyteDocker.outputs)
 }
 
+// used to allow local iteration to work
```
Comment (keu) [inline airbyte-integrations/connectors/source-google-adwords-singer/build.gradle]: this code smells, why do we need it?

**Review by keu — CHANGES_REQUESTED** [review]

File: airbyte-integrations/connectors/source-google-adwords-singer/build.gradle (hunk 1 above)
Comment (Zirochkaa) [reply airbyte-integrations/connectors/source-google-adwords-singer/build.gradle]: Don't know, I'm not the author of this.

Comment (Zirochkaa) [reply airbyte-integrations/connectors/source-google-adwords-singer/build.gradle]: I just reverted changes that were there when I took this issue.

**Pull request closed without merging**